CMIP7/esm1p6/spinup/atmosphere/land/generate_restart_from_vegetation_distribution/run_remap.sh

```diff
@@ -0,0 +1,24 @@
+#PBS -P rp23
+#PBS -N remap_veg
+#PBS -q normal
+#PBS -l walltime=2:00:00
+#PBS -l mem=8GB
+#PBS -l ncpus=1
+#PBS -l storage=gdata/rp23+gdata/xp65+gdata/p66+gdata/vk83+gdata/access
+#PBS -l wd
+
+module use /g/data/xp65/public/modules
+module load conda/analysis3
```

Comment:
I ended up having to specify a specific version as newer ones didn't have mule. I think there's work being done on including mule in newer versions, but I the following worked for me for the moment:
```suggestion
module load conda/analysis3-24.0
```